projects/core/src/occ/adapters/checkout/occ-checkout-delivery.adapter.ts

```diff
@@ -99,7 +99,7 @@ export class OccCheckoutDeliveryAdapter implements CheckoutDeliveryAdapter {
         this.getCartEndpoint(userId) + cartId + '/deliverymodes'
       )
       .pipe(
-        catchError((error: any) => throwError(error.json())),
+        catchError(error => throwError(makeHttpErrorSerializable(error))),
```

Comment:
I would opt for not using `makeHttpErrorSerializable` in adapter.

Reply:
Good catch, reverted.